### PR TITLE
Swerve calib

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/robot/JoysticksBindings.java [hunk 1]
```diff
@@ -24,19 +25,32 @@ public static void configureBindings(Robot robot) {
...
+		
 		usedJoystick.A.onTrue(robot.getSuperstructure().setState(RobotState.IDLE));
 		usedJoystick.B.onTrue(robot.getSuperstructure().setState(RobotState.INTAKE));
 		usedJoystick.START.onTrue(robot.getSuperstructure().setState(RobotState.TRANSFER_ELEVATOR_SHOOTER));
 		usedJoystick.BACK.onTrue(robot.getSuperstructure().setState(RobotState.TRANSFER_SHOOTER_ELEVATOR));
 		usedJoystick.Y.onTrue(robot.getSuperstructure().setState(RobotState.INTAKE_OUTTAKE));
 		usedJoystick.X.onTrue(robot.getSuperstructure().setState(RobotState.SPEAKER));
 		usedJoystick.POV_UP.onTrue(robot.getSuperstructure().setState(RobotState.AMP));
-		usedJoystick.POV_DOWN.onTrue(robot.getSuperstructure().setState(RobotState.PRE_AMP));
+		usedJoystick.POV_DOWN.onTrue(robot.getSuperstructure().setState(RobotState.PRE_AMP));;
```
Comment (drorta) [inline src/main/java/frc/robot/JoysticksBindings.java]: What

File: src/main/java/frc/robot/JoysticksBindings.java [hunk 2]
```diff
@@ -24,19 +25,32 @@ public static void configureBindings(Robot robot) {
...
+		
 	}
 
 	private static void secondJoystickButtons(Robot robot) {
 		SmartJoystick usedJoystick = SECOND_JOYSTICK;
+		
+		
+		// bindings...
+
```
Comment (drorta) [inline src/main/java/frc/robot/JoysticksBindings.java]: Fix spaces

File: src/main/java/frc/robot/constants/IDs.java [hunk 3]
```diff
@@ -15,7 +15,7 @@ public class IDs {
 		PowerDistribution.ModuleType.kRev
 	);
 
-	public static final Phoenix6DeviceID PIGEON_2 = new Phoenix6DeviceID(0, BusChain.CANIVORE);
+	public static final Phoenix6DeviceID PIGEON_2 = new Phoenix6DeviceID(0, BusChain.ROBORIO);
```
Comment (hillelv) [inline src/main/java/frc/robot/constants/IDs.java:18]: Use the second constructor

File: src/main/java/frc/robot/constants/IDs.java
Comment (hillelv) [inline src/main/java/frc/robot/constants/IDs.java:1]: Why the id numbers are so מצחיק

File: src/main/java/frc/robot/subsystems/swerve/factories/modules/encoder/EncoderFactory.java [hunk 4]
```diff
@@ -1,24 +1,17 @@
...
-		return switch (Robot.ROBOT_TYPE) {
-			case REAL -> switch (modulePosition) {
-				case FRONT_LEFT -> EncoderRealConstants.generateEncoderStuff(logPath, IDs.CANCodersIDs.FRONT_LEFT_ENCODER);
-				case FRONT_RIGHT -> EncoderRealConstants.generateEncoderStuff(logPath, IDs.CANCodersIDs.FRONT_RIGHT_ENCODER);
-				case BACK_LEFT -> EncoderRealConstants.generateEncoderStuff(logPath, IDs.CANCodersIDs.BACK_LEFT_ENCODER);
-				case BACK_RIGHT -> EncoderRealConstants.generateEncoderStuff(logPath, IDs.CANCodersIDs.BACK_RIGHT_ENCODER);
-			};
-			case SIMULATION -> null;// TODO
-		};
+		return new EncoderStuff(new EmptyAngleEncoder(logPath), new SuppliedAngleSignal("angle", () -> 0.0, AngleUnit.ROTATIONS));
```
Comment (drorta) [inline src/main/java/frc/robot/subsystems/swerve/factories/modules/encoder/EncoderFactory.java:14]: Why? No encoders?

Comment (hillelv) [reply src/main/java/frc/robot/subsystems/swerve/factories/modules/encoder/EncoderFactory.java:14]: No encoders on the new robot

Comment (drorta) [reply src/main/java/frc/robot/subsystems/swerve/factories/modules/encoder/EncoderFactory.java:14]: Damn

**Pull request merged**